ocaml/sdk-gen/component-test/jsonrpc-server/server.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+from aiohttp import web
+
+
+def load_json_files():
+    """
+    Load all json files from spec directory
+    """
+    data = {}
+    for filename in os.listdir("spec/"):
+        if filename.endswith(".json"):
+            with open(f"spec/{filename}", "r") as f:
```

Comment:
can these warnings be fixed?

Reply:
Only E0401 can't be fixed

Reply:
Please, silence the warning using adding `  # pytype: disable=import-error` at the end of the line